go/cmd/server/logging/logging_test.go

```diff
@@ -0,0 +1,262 @@
+package logging
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/apache/arrow/go/v8/arrow"
+	"github.com/apache/arrow/go/v8/arrow/array"
+	"github.com/feast-dev/feast/go/internal/feast/model"
+	"github.com/feast-dev/feast/go/internal/test"
+	"github.com/feast-dev/feast/go/protos/feast/serving"
+	"github.com/feast-dev/feast/go/protos/feast/types"
+	gotypes "github.com/feast-dev/feast/go/types"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+func TestLoggingChannelTimeout(t *testing.T) {
+	// Pregenerated using `feast init`.
+	loggingService, err := NewLoggingService(nil, 1, "", false)
+	assert.Nil(t, err)
+	assert.Empty(t, loggingService.memoryBuffer.logs)
+	ts := timestamppb.New(time.Now())
+	newLog := Log{
+		FeatureStatuses: []serving.FieldStatus{serving.FieldStatus_PRESENT},
+		EventTimestamps: []*timestamppb.Timestamp{ts, ts},
+	}
+	loggingService.EmitLog(&newLog)
+	// Wait for memory buffer flush
+	assert.Eventually(t, func() bool { return true }, 100*time.Millisecond, DEFAULT_LOG_INSERT_TIMEOUT)
```

Comment:
what is being checked here if it always true?
`assert.Eventually(t, func() bool { return true }`

Reply:
Fixed.